### PR TITLE
LRQA-68305, LRQA-68306 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-68305
https://issues.liferay.com/browse/LRQA-68306

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:relevant

**Comment by melodywu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 1]
```diff
@@ -2901,6 +2901,19 @@ definition {
...
+		var key_userFirstLastName = "${userFirstLastName}";
+		var key_userScreenName = "${userScreenName}";
+
+		AssertTextEquals(
+			locator1 = "UsersAndOrganizations#USER_LIST_FISTLAST_NAME",
+			value1 = "${userFirstLastName}");
+
+		AssertTextEquals(
+			locator1 = "UsersAndOrganizations#USER_LIST_SCREEN_NAME",
+			value1 = "${userScreenName}");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/User.macro:2927]: Generic paths from LexiconList should be able to do the same

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 2]
```diff
@@ -2901,6 +2901,19 @@ definition {
...
+		var key_userFirstLastName = "${userFirstLastName}";
+		var key_userScreenName = "${userScreenName}";
+
+		AssertTextEquals(
+			locator1 = "UsersAndOrganizations#USER_CARDS_FISTLAST_NAME",
+			value1 = "${userFirstLastName}");
+
+		AssertTextEquals(
+			locator1 = "UsersAndOrganizations#USER_CARDS_SCREEN_NAME",
+			value1 = "${userScreenName}");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/User.macro:2914]: Generic paths from Cards should be able to do the same

**Comment by patriciajeanperez** [comment]
Looks very good, @melodywu! Just make sure we try to use generic paths first before creating specific ones. Don't forget to rebase as well to only show necessary commits to review. Thank you!

**Comment by patriciajeanperez** [comment]
ci:close

**Pull request closed without merging**